docs/source/nlp/tutorials/quickstarts/plot_token_classification.py

```diff
@@ -0,0 +1,175 @@
+classification tasks. For a more complete example showcasing the range of checks and capabilities of the NLP package,
+refer to our :ref:`Multiclass Quickstart <nlp__multiclass_quickstart>`. We will cover the following steps:
+
+1. `Creating a TextData object and auto calculating properties <#setting-up>`__
+2. `Running checks <#running-checks>`__
+
+To run deepchecks for NLP, you need the following for both your train and test data:
+
+1. Your :ref:`text data <nlp__textdata_object>` - a list of strings, each string is a single sample
+   (can be a sentence, paragraph, document, etc.).
+2. Your labels - either a :ref:`Text Classification <nlp_supported_text_classification>` label or a
+   :ref:`Token Classification <nlp_supported_token_classification>` label. These are not needed for checks that
+   don't require labels (such as the Embeddings Drift check or most data integrity checks), but are needed for
+   many other checks.
+3. Your model's predictions (see :ref:`nlp__supported_tasks` for info on supported formats). These are needed only for
+   the model related checks, shown in the `Model Evaluation <#model-evaluation>`__ section of this guide.
+
+If you don't have deepchecks installed yet:
+
+.. code:: python
+
+    import sys
+    !{sys.executable} -m pip install deepchecks[nlp] -U --quiet #--user
+
+Some properties calculated by ``deepchecks.nlp`` require additional packages to be installed. You can
+install them by running:
+
+.. code:: python
+
+    import sys
+    !{sys.executable} -m pip install [nlp-properties] -U --quiet #--user
+
+Setting Up
+==========
+
+Load Data
+---------
+For the purpose of this guide, we'll use a small subset of the
+`SCIERC <http://nlp.cs.washington.edu/sciIE/>`__ dataset:
+
+"""
+from pprint import pprint
+from deepchecks.nlp import TextData
+from deepchecks.nlp.datasets.token_classification import scierc_ner
+
+train, test = scierc_ner.load_data(data_format='Dict')
+pprint(train['text'][0][:10])
+pprint(train['label'][0][:10])
+
+# %%
+#
+# The SCIERC dataset is a dataset of scientific articles with annotations for named entities, relations and coreference.
+# In this example we'll only use the named entities annotations, which are the labels we'll use for our token
+# classification task.
+# We can see that we have the article text itself, and the labels for each token in the text in the
+# :ref:`IOB format <nlp__supported_token_classification>`.
+#
+# Create a TextData Objects
+# -------------------------
+#
+# We can now create a :ref:`TextData <nlp__textdata_object>` object for the train and test dataframes.
+# This object is used to pass your data to the deepchecks checks.
+#
+# To create a TextData object, the only required argument is the tokenized text itself, but passing only the text
+# will prevent multiple checks from running. In this example we'll pass the label and define the task type.
```

Comment:
I wrote that part of the code and I'm not sure I get what you're saying here.
I think this is the place to explain the advantages and disadvantages of raw_text vs tokenized_text, the automatic inference, and maybe a word on the format?

Reply:
What I'm actually doing here is telling them that label is not required.
Also for token classification tokenized_text is mandatory, but will try to elaborate.